tests/test_edf2bids.py

```diff
@@ -89,6 +89,46 @@ def test_edf_nan_in_tsv(eyelink_test_data_dir):
     assert count == 0
 
 
+@pytest.mark.skipif(not _check_edf2asc_present(), reason="edf2asc missing")
+def test_number_columns_2eyes_tsv(eyelink_test_data_dir):
+    """Check that values for both eyes were extracted by number of columns (function _samples_to_data_frame)"""
+    input_dir = eyelink_test_data_dir / "2eyes"
```

Comment:
the 2eyes folder has no asc files, and that's why this test fails

if you want test some things on the asc file of a particular dataset we have on OSF make sure the asc file is on OSF

Reply:
I modified the folder with the testfiles on OSF. Here's what I did:

- each folder now has 2 asc files as it is done by our converter: one _samples.asc and one _events.asc file 
- subsequently, I deleted the old asc files
- added an edf file to the "rest" folder
- added a wiki which gives info on how the edf file was turned into the two asc files 
- deleted the "decision" dataset as the edf file could not be converted and something seemed wrong with the edf file

Reply:
thanks @julia-pfarr 
will quickly edit the test utilities to adapt